packages/lab/src/slider/internal/SliderTrack.css

```diff
@@ -0,0 +1,123 @@
+.saltSliderTrack {
+  --slider-progressPercentage: "0%";
+  /* For range */
+  --slider-progressPercentageStart: "0%";
+  --slider-progressPercentageEnd: "0%";
+
+  --slider-marker-percentage: "0%";
+  --slider-track-background: var(--salt-track-borderColor);
+  --slider-track-fill: var(--salt-accent-borderColor);
+
+  display: flex;
+  position: relative;
+  width: 100%;
+}
+
+.saltSliderTrack-withMarkers {
+  justify-content: unset;
+}
+
+.saltSliderTrack-disabled {
+  cursor: not-allowed;
+}
+
+.saltSliderTrack-container {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.saltSliderTrack-wrapper {
+  align-items: center;
+  display: flex;
+  width: 100%;
+  height: var(--salt-size-base);
+}
+
+.saltSliderTrack-rail {
+  border-radius: var(--salt-palette-corner-strongest);
+  flex: 100%;
+  order: 0;
+  position: relative;
+  width: 100%;
+  background: linear-gradient(
+    to right,
+    var(--slider-track-fill) calc(var(--slider-progressPercentage) - 1px),
```

Comment:
Are all these `1px` related? Should we save this in a variable to make it easier to understand